ios/FluentUI/Core/Theme/Tokens/ControlTokenSet.swift

```diff
@@ -53,6 +53,22 @@ public class ControlTokenSet<T: TokenSetKey>: ObservableObject {
         }
     }
 
+    /// Convenience method to override multiple tokens from another token set.
+    ///
+    /// This is useful if `otherTokenSet` belongs to a parent control and `self` belongs to a child control.
+    ///
+    /// - Parameter otherTokenSet: The token set we will be pulling values from.
+    /// - Parameter mapping: A `Dictionary` that maps our own tokens that we wish to override with
+    /// their corresponding tokens in `otherTokenSet`.
+    public func setOverrides<U>(from otherTokenSet: ControlTokenSet<U>, mapping: [T: U]) {
```

Comment:
Is this a safe addition? With our current token philosophy we want to add subcontrol tokens in the parent control and any overrides should happen from the parent. Wouldn't this open us up to potentially strange behaviors in situations where overrides occur from both parent and child?

Reply:
Do we have a preferred way to pass tokens to child controls? I don't see any examples of this currently in this repo.

Reply:
Using [this bit of code](https://github.com/microsoft/fluentui-apple/blob/b7e84b4137ab808239940b2b37892285ca5f41e9/ios/FluentUI/Tab%20Bar/SideTabBar.swift#L381) as a reference point, I think what we really want here is to replace the RHS with `otherTokenSet.overrideValue(forToken: otherToken)`.